news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md

```diff
@@ -0,0 +1,207 @@
+* [#11690](https://github.com/PrestaShop/PrestaShop/pull/11690): Fix install_upgrade campaign. Thank you [@ansar21mallouli](https://github.com/ansar21mallouli)
+
+
+## Code changes in modules, themes & tools
+
+### Google Sitemap
+
+* [#71](https://github.com/PrestaShop/gsitemap/pull/71): Minor improvements. Thank you [@MathiasReker](https://github.com/MathiasReker)
+
+
+### Products in the same category
+
+* [#6](https://github.com/PrestaShop/ps_categoryproducts/pull/6): Wrong cache key used in ps_categoryproducts. Thank you [@Matt75](https://github.com/Matt75)
+* [#8](https://github.com/PrestaShop/ps_categoryproducts/pull/8): Deploying v1.0.4 of ps_categoryproducts, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Link list
+
+* [#57](https://github.com/PrestaShop/ps_linklist/pull/57): Fix update when table cms_block_page does not exist, by [@jolelievre](https://github.com/jolelievre)
+* [#59](https://github.com/PrestaShop/ps_linklist/pull/59): Deploy v3.0.2 of ps_linklist, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### MBO
+
+* [#76](https://github.com/PrestaShop/ps_mbo/pull/76): Character issue, by [@MrBaiame](https://github.com/MrBaiame)
+* [#77](https://github.com/PrestaShop/ps_mbo/pull/77): Merge fixes to master, by [@jolelievre](https://github.com/jolelievre)
+* [#78](https://github.com/PrestaShop/ps_mbo/pull/78): Update version to 1.1.1, by [@jolelievre](https://github.com/jolelievre)
+* [#79](https://github.com/PrestaShop/ps_mbo/pull/79): Update version to 1.1.1, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Catalog evalutation
+
+* [#10](https://github.com/PrestaShop/statscheckup/pull/10): Fix notice due to missing translation key. Thank you [@Matt75](https://github.com/Matt75)
+
+
+### Visits and Visitors
+
+* [#3](https://github.com/PrestaShop/statsvisits/pull/3): Fix error 500 on Export CSV. Thank you [@Matt75](https://github.com/Matt75)
+
+
+### Visitors online
+
+* [#2](https://github.com/PrestaShop/statslive/pull/2): getCustomersOnline(): fix SQL clauses to be able to use index. Thank you [@Kioob](https://github.com/Kioob)
+
+
+### GDPR
+
+* [#10](https://github.com/PrestaShop/psgdpr/pull/10): Add export feature + update translations + fix translation issue with csv and more. Thank you [@Joukz](https://github.com/Joukz)
+
+
+### PrestaFraud
+
+* [#11](https://github.com/PrestaShop/prestafraud/pull/11): Replace Forge URL by GitHub issues, by [@eternoendless](https://github.com/eternoendless)
+
+
+### Docker internal images
+
+* [#19](https://github.com/PrestaShop/docker-internal-images/pull/19): Expose HTTPS port, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+## Changes in Documentation
+
+* [#135](https://github.com/PrestaShop/docs/pull/135): Update to Grid docs. Thank you [@sarjon](https://github.com/sarjon)
+* [#168](https://github.com/PrestaShop/docs/pull/168): Add a paragraph in 1.7.5 changes page about the _legacy_link, by [@jolelievre](https://github.com/jolelievre)
```

Comment:
Will `_legacy_link` create display issues on the core weekly ? Will markdown consider it styling ?

If yes you can rename it "legacy_link" or "legacy link" ;)

Reply:
I can also escape if this is not working well when merged. Unfortunately, the markdown preview of GitHub is not exactly the same than the Jenkins build.
With jenkins, I usually have issues with |.